packages/react-components/src/organisms/Toast.tsx

```diff
@@ -72,21 +80,38 @@ const crossPlaceholderStyles = css(iconStyles, {
 const wrapStyles = css({
   display: 'flex',
   flexWrap: 'wrap',
+  flexFlow: 'row',
 });
 
+export const getIcon = (accent?: 'alert' | 'info'): EmotionJSX.Element => {
```

Comment:
Wouldn't this be better expressed as a Record and set a default accent in the props?